src/components/BlankSlate/BlankSlate.css.js

```diff
@@ -0,0 +1,57 @@
+import { getColor } from '../../styles/utilities/color'
+import variableFontSize from '../../styles/utilities/variableFontSize'
+import styled from '../styled'
+
+import Heading from '../Heading'
+import Text from '../Text'
+import Illo from '../Illo'
+
+export const IlloUI = styled(Illo)`
+  padding-bottom: 18px;
+  margin: 0;
+  align-self: center;
+`
+
+export const HeadingUI = styled(Heading)`
+  &.is-h3 {
+    ${variableFontSize({ fontSize: 18 })};
+  }
+  color: ${getColor('charcoal.800')};
+`
+
+export const TextUI = styled(Text)`
+  color: ${getColor('charcoal.300')};
+  padding-top: 8px;
+
+  b,
+  strong {
+    color: ${getColor('charcoal.500')};
+  }
+`
+
+export const BlankSlateUI = styled('div')`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  text-align: center;
+  flex-direction: column;
+  padding: 56px 0 68px;
```

Comment:
For the container we might want to add some padding on the sides too, to avoid these scenarios:

<img width="394" alt="Storybook 2019-04-05 13-09-47" src="https://user-images.githubusercontent.com/1414086/55623823-6018c580-57a4-11e9-9d58-915c054d85f7.png">

Reply:
nice catch, I'll update the padding.